src/parser/ui/DistanceRadarRing.tsx

```diff
@@ -0,0 +1,29 @@
+import { CSSProperties } from '@emotion/serialize';
```

Comment:
Might want the `CSSProperties` from React instead of emotion

Reply:
Oops, yup that's a bad auto import